dbus-service.py

```diff
@@ -92,7 +92,7 @@ def impl_DownloadSubscription(self, invocation, parameters):
         subscription_id, = parameters.unpack()
```

Comment:
Most of the code already supported Python 3 -- fixing up the remaining prints should be enough without the other stuff.